docs/general/administration/migrate.md

```diff
@@ -5,9 +5,15 @@ title: Migrating
 
 # Migrating
 
-It is possible to migrate your system to another system by using environment variables.
-It's possible to do this via the command line or by using Docker environment variables.
-To read more, see the [Configuration](/docs/general/administration/configuration) page.
+This page explains how to migrate from Emby to Jellyfin as well as how to migrate a Jellyfin installation.
+
+Jellyfins internal databases are not designed for migrations. Depending on your case there may be ways to work around this, f.ex. by just migrating parts of the data, or because it's possible maintain the same file paths.
```

Comment:
I would write out "for example" instead of abbreviating it.

Reply:
Okay, will change it together with the other changes.